mmv1/templates/terraform/examples/gkehub_scope_rbac_custom_role_binding_basic.tf.tmpl

```diff
@@ -0,0 +1,26 @@
+resource "google_gke_hub_scope" "scope" {
+  scope_id = "tf-test-scope%{random_suffix}"
+}
+
+resource "google_gke_hub_feature" "rbacrolebindingactuation" {
+  name = "rbacrolebindingactuation"
+  location = "global"
+  spec {
+    rbacrolebindingactuation {
+      allowed_custom_roles = ["my-custom-role"]
+    }
+  }
+}
+
+resource "google_gke_hub_scope_rbac_role_binding" "{{$.PrimaryResourceId}}" {
+  scope_rbac_role_binding_id = "tf-test-scope-rbac-role-binding%{random_suffix}"
+  scope_id = google_gke_hub_scope.scope.scope_id
```

Comment:
this dependency may allow you to skip the depends_on?

or maybe only if its an output?